src/app/common/vtabs.component.ts

```diff
@@ -24,7 +24,8 @@ import { IsWAC } from 'environments/environment';
                         [ngClass]="{active: tab.active}"
                         (keyup.space)="selectItem(tab)"
                         (keyup.enter)="selectItem(tab)"
-                        (click)="selectItem(tab)">
+                        (focus)="selectItem(tab)"
+                        (click)=" IsWAC? selectItem(tab): '' ">
```

Comment:
We should keep the behavior the same for WAC and site

Reply:
Done.